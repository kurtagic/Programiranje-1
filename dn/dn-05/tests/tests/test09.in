55
7 81
7 80
7 77
7 56
7 55
7 86
7 68
7 75
7 60
7 73
7 79
7 55
7 46
7 75
7 68
7 59
7 58
7 63
7 63
7 75
7 71
7 65
7 57
7 75
7 60
7 45
7 61
7 47
7 68
7 49
7 75
7 38
7 54
7 81
7 70
7 52
7 54
7 74
7 74
7 52
7 65
7 85
7 38
7 78
7 74
7 41
7 67
7 70
7 74
7 61
7 77
7 51
7 52
7 73
7 66
