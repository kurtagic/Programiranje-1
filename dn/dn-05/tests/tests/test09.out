38: 6
39: 4
41: 3
42: 1
45: 6
46: 4
47: 4
48: 2
49: 3
50: 1
51: 9
52: 10
53: 8
54: 15
55: 13
56: 7
57: 12
58: 10
59: 7
60: 7
61: 11
62: 9
63: 6
64: 4
65: 6
66: 15
67: 13
68: 10
69: 7
70: 9
71: 7
72: 1
73: 30
74: 28
75: 25
76: 13
77: 15
78: 13
79: 10
80: 7
81: 7
82: 5
85: 6
86: 4
87: 1
